src/hotspot/share/gc/g1/g1CardSet.cpp

```diff
@@ -240,12 +241,15 @@ void G1CardSetCoarsenStats::print_on(outputStream* out) {
 
 class G1CardSetHashTable : public CHeapObj<mtGCCardSet> {
   using ContainerPtr = G1CardSet::ContainerPtr;
+  using CHTScanTask = CardSetHash::ScanTask;
 
+  const static uint BucketClaimSize = 16;
```

Comment:
Some comment on how `16` is derived would be nice and probably useful for future re-evaluation.